src/state/CallViewModel.ts

```diff
@@ -464,16 +465,36 @@ export class CallViewModel extends ViewModel {
       },
     );
 
+  /**
+   * Observable for changes to the MatrixRTCSession membership list.
+   *
+   * We do this to ensure that we only listen once to the event and then share internally.
+   */
+  private readonly membershipsChanged$ = fromEvent(
+    this.matrixRTCSession,
+    MatrixRTCSessionEvent.MembershipsChanged,
+  ).pipe(share());
+
+  /**
+   * Observable for changes to the Matrix Room member data.
+   *
+   * We do this to ensure that we only listen once to the event and then share internally.
+   */
+  private readonly roomMembers$ = fromEvent(
+    this.matrixRTCSession.room,
+    RoomStateEvent.Members,
+  ).pipe(share());
```

Comment:
Any time we manually create a hot observable with `share`, we should also pipe it through `this.scope.bind` to ensure that its execution will terminate when the ViewModel is destroyed.